src/bracuganda/AndroidManifest.xml

```diff
@@ -1,7 +1,6 @@
 <?xml version="1.0" encoding="utf-8"?>
 <manifest xmlns:android="http://schemas.android.com/apk/res/android"
-    xmlns:tools="http://schemas.android.com/tools"
-    package="org.medicmobile.webapp.mobile">
```

Comment:
No longer need to specify this `package` since we set the `namespace` in the build.gradle.  (This was one of the changes that automatically happened when I upgraded the Android Gradle Plugin.)